### PR TITLE
[FEATURE] Icon size can be set individually. 

### PR DESCRIPTION
**Pull request by monosize**
If you want to use bigger icons for the editorial users.

### CODE REVIEW

File: Classes/Service/ConfigurationService.php [hunk 1]
```diff
@@ -86,6 +81,10 @@ public function injectRecordService(WorkspacesAwareRecordService $recordService)
 	 */
 	public function __construct() {
 		$this->defaultIcon = '../' . ExtensionManagementUtility::siteRelPath('fluidcontent') . 'Resources/Public/Icons/Plugin.png';
+
+		$this->extConf = unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['fluidcontent']);
+		$this->extConf['iconWidth'] = $this->extConf['iconWidth']?:'';
```
Comment (NamelessCoder) [inline Classes/Service/ConfigurationService.php]: There is only one other thing to remark: please preserve the constants and use those as default values for the alternative case in these ternary conditions. That will allow the site to show the default 24x24 sizes until the site administrator saves the EM config. E.g. `$this->extConf['iconWidth'] ? : self::ICON_WIDTH;` (note spaces for CGL compliance).

**Comment by xf-** [comment]
I'm no fan of making this one flexible.
Normal icon in list or page module is max 16x16 and for example 128x128 resized to 16x16px sizes looks not so good.
One of our last changes was to render Icons 24x24px to have same size as standard elements and same list.

File: Classes/Service/ConfigurationService.php [hunk 2]
```diff
@@ -330,7 +329,9 @@ protected function buildWizardTabItem($tabId, $id, $form, $templateFileIdentity)
 		}
 
 		if (TRUE === file_exists($icon)) {
-			$icon = '../..' . MiscellaneousUtility::createIcon($icon, self::ICON_WIDTH, self::ICON_HEIGHT);
+			$width = $this->extConf['iconWidth'];
```
Comment (xf-) [inline Classes/Service/ConfigurationService.php]: why set a small readable variable to another variable?

File: ext_conf_template.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# cat=basic/enable; type=integer; label=Set the width  of the content element icon. 0 = no scaling
```
Comment (xf-) [inline ext_conf_template.txt]: LLL is supported as label

File: .gitignore [hunk 4]
```diff
@@ -1,3 +1,4 @@
 build/logs
 vendor
-.idea
\ No newline at end of file
+.idea
+.iml
```
Comment (xf-) [inline .gitignore]: has nothing to-do with this commit.

**Comment by monosize** [comment]
The constants are now preserved and the code was simplified.
 `.gitignore` was removed from the commit.

**Comment by NamelessCoder** [comment]
Speaking only about CGL, I'm okay with this one. Regarding whether or not the feature is desired, there are a few other people's opinions we'll need.

**Comment by cedricziel** [comment]
+1, Accessibility never hurts - please rebase :)

**Comment by NamelessCoder** [comment]
Manually picked, rebased, pushed - thanks @monosize!

**Pull request closed without merging**